### DISCUSSION TITLE
ignore builds when running `goreleaser release`

### DISCUSSION THREAD

**Discussion by till** [goreleaser/goreleaser#5041] [Q&A] [answered]
Is there a way to ignore certain GOOS/GOARCH combinations when we run `goreleaser release`?

For context, we have different OS' that we work and develop on, but production is almost always linux/amd64 — so I'd like to disable arm variants for Linux/OSX etc.. When creating a release.

**Comment by caarlos0** [comment]
I think you can probably leave just the production build in the yaml, and then build locally with `goreleaser build --single-target --snapshot --clean`

**Reply by till** [reply, marked as answer]
Been trying this and a few other options. We currently use: `goreleaser r --snapshot --clean --skip=publish`

To build the binary and build the docker image. I think I need multiple config files.

Btw, is there a place to talk about the Pro pricing? I saw it might do what I need, but unsure how pricing applies to us.

**Reply by caarlos0** [reply]
hey!

yes, if you wanna build docker images too I'm afraid you might need two different configs.

> Btw, is there a place to talk about the Pro pricing? 

Sure, you can ping me on discord (https://goreleaser.com/discord) or email (carlos at becker dot software)
